style.css

```diff
@@ -0,0 +1,39 @@
+* {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+}
+
+.container {
+  padding-top: 50px;
+  width: 100%;
+  height: 100%;
+  display: grid;
+  grid-template-columns: 20% 80%;
+  background-color: #eee;
+}
+
+.sideNav {
+  height: 100vh;
+  padding: 1rem;
+}
+
+.sideNav h1 {
+  text-transform: capitalize;
+}
+
+.sideNav ul {
+  list-style-type: none;
+  width: 100%;
+  overflow: scroll;
+}
+
+.sideNav ul li {
+  border-bottom: 1px solid #000;
+  padding: 1.2rem;
+}
+
+.sideNav ul li a {

```

Comment:
same is the case with it like above
.sideNav ul > li > a

Don't worry, these are the conventions and you're going to use to it.

Great Job BTW, I really appreciate your efforts and hard work. KEEP IT UP  :blush: